### ISSUE TITLE
Use _.runInContext

### ISSUE THREAD

**Issue by jdalton** [overlookmotel/overlook-utils#1]
When mixing in things for the node package of Lo-Dash you should use the `_.runInContext()` method to create a pristine instance of Lo-Dash that can have mixins added without modifying the `lodash` package instance.

``` js
var _ = require('lodash').runInContext();
// then
_.mixin(...);
// and
module.exports = _;
```

**Comment by overlookmotel** [comment]
Thanks for bringing this to my attention. Fixed in v0.0.7 (not that anyone else will be using this module anyway - at this point it's just for my private use and I've only put it on npm for ease of including from my other projects)

**Comment by overlookmotel** [comment]
By the way, thank you so much for creating lodash, @jdalton. I use it every day!
